libraries/ms-helper/src/x86/adc_periodic_reader.c

```diff
@@ -0,0 +1,82 @@
+#include "adc_periodic_reader.h"

```

Comment:
is this any different than the stm version?

Reply:
Yup, I need to seperate them since ADC doesent work on x86